java/c/src/main/java/org/apache/arrow/c/BufferImportTypeVisitor.java

```diff
@@ -210,9 +210,38 @@ public List<ArrowBuf> visit(ArrowType.Utf8 type) {
     }
   }
 
+  private List<ArrowBuf> visitVariableWidthView(ArrowType type) {
+    final int viewBufferIndex = 1;
+    try (ArrowBuf view = importFixedBytes(type, viewBufferIndex, BaseVariableWidthViewVector.ELEMENT_SIZE)) {
```

Comment:
Why is view being treated specially here? Shouldn't it be the size buffer?

Reply:
Treated specially means? Sorry I didn't follow.

Reply:
Why is the view buffer the first one we import? The one we care about is the sizes buffer.

Reply:
Ah nothing special there, I can change the order. Validity buffer, view buffer, sizes buffer then variadic buffer. 
Any issue doing it this way?

Reply:
In the sense that...you need the sizes buffer to do anything, so import that first, then discard it at the end? Why are we treating the view buffer specially when it's not special here?

Reply:
I updated, is it okay?